modules/programs/chromium.nix

```
@@ -156,6 +160,7 @@ in {
     google-chrome-dev =
       browserModule pkgs.google-chrome-dev "Google Chrome Dev" false;
     brave = browserModule pkgs.brave "Brave Browser" false;
+    vivaldi = browserModule pkgs.brave "Vivaldi Browser" false;
```

Are you certain this doesn't have to be something like
```suggestion
    vivaldi = browserModule pkgs.vivaldi "Vivaldi Browser" false;
```
?